### ISSUE TITLE
test failure (FormatterTests) on 2.12.7 nightly build 

### ISSUE THREAD

**Issue by SethTisue** [lihaoyi/utest#168]
this turned up in the Scala 2.12 community build

with

```
resolvers += "scala-integration" at
 "https://scala-ci.typesafe.com/artifactory/scala-integration/"
```

then `++2.12.7-bin-bd76826` followed by `utestJVM/test`

```
X test.utest.FormatterTests.wrapped 33ms 
  utest.AssertionError: assert(trimmedOutput == expected)
  trimmedOutput: String = X MyTestSuite.test1
    java.lang.Exception: wrapper
      test.utest.FormatterTests$.liftedTree1$1(Forma
      tterTests.scala:19)
      test.utest.FormatterTests$.$anonfun$tests$3(Fo
      rmatterTests.scala:17)
    utest.AssertionError: try assert(x == 2)
    x: Int = 1
      utest.asserts.Asserts$.assertImpl(Asserts.scal
      a:114)
      test.utest.FormatterTests$.liftedTree1$1(Forma
      tterTests.scala:17)
      test.utest.FormatterTests$.$anonfun$tests$3(Fo
      rmatterTests.scala:17)
  + MyTestSuite.test2  1
  X MyTestSuite.test3
    java.lang.IndexOutOfBoundsException: 10
      scala.collection.LinearSeqOptimized.apply(Line
      arSeqOptimized.scala:63)
      scala.collection.LinearSeqOptimized.apply$(Lin
      earSeqOptimized.scala:61)
      scala.collection.immutable.List.apply(List.sca
      la:85)
      test.utest.FormatterTests$.$anonfun$tests$6(Fo
      rmatterTests.scala:26)
  - MyTestSuite
    X test1
      java.lang.Exception: wrapper
        test.utest.FormatterTests$.liftedTree1$1(For
        matterTests.scala:19)
        test.utest.FormatterTests$.$anonfun$tests$3(
        FormatterTests.scala:17)
      utest.AssertionError: try assert(x == 2)
      x: Int = 1
        utest.asserts.Asserts$.assertImpl(Asserts.sc
        ala:114)
        test.utest.FormatterTests$.liftedTree1$1(For
        matterTests.scala:17)
        test.utest.FormatterTests$.$anonfun$tests$3(
        FormatterTests.scala:17)
    + test2  1
    X test3
      java.lang.IndexOutOfBoundsException: 10
        scala.collection.LinearSeqOptimized.apply(Li
        nearSeqOptimized.scala:63)
        scala.collection.LinearSeqOptimized.apply$(L
        inearSeqOptimized.scala:61)
        scala.collection.immutable.List.apply(List.s
        cala:85)
        test.utest.FormatterTests$.$anonfun$tests$6(
        FormatterTests.scala:26)
  expected: String = X MyTestSuite.test1
    java.lang.Exception: wrapper
      test.utest.FormatterTests$.liftedTree1$1(Forma
      tterTests.scala:19)
      test.utest.FormatterTests$.$anonfun$tests$3(Fo
      rmatterTests.scala:17)
    utest.AssertionError: try assert(x == 2)
    x: Int = 1
      utest.asserts.Asserts$.assertImpl(Asserts.scal
      a:114)
      test.utest.FormatterTests$.liftedTree1$1(Forma
      tterTests.scala:17)
      test.utest.FormatterTests$.$anonfun$tests$3(Fo
      rmatterTests.scala:17)
  + MyTestSuite.test2  1
  X MyTestSuite.test3
    java.lang.IndexOutOfBoundsException: 10
      scala.collection.LinearSeqOptimized.apply(Line
      arSeqOptimized.scala:63)
      scala.collection.LinearSeqOptimized.apply$(Lin
      earSeqOptimized.scala:61)
      scala.collection.immutable.List.apply(List.sca
      la:86)
      test.utest.FormatterTests$.$anonfun$tests$6(Fo
      rmatterTests.scala:26)
  - MyTestSuite
    X test1
      java.lang.Exception: wrapper
        test.utest.FormatterTests$.liftedTree1$1(For
        matterTests.scala:19)
        test.utest.FormatterTests$.$anonfun$tests$3(
        FormatterTests.scala:17)
      utest.AssertionError: try assert(x == 2)
      x: Int = 1
        utest.asserts.Asserts$.assertImpl(Asserts.sc
        ala:114)
        test.utest.FormatterTests$.liftedTree1$1(For
        matterTests.scala:17)
        test.utest.FormatterTests$.$anonfun$tests$3(
        FormatterTests.scala:17)
    + test2  1
    X test3
      java.lang.IndexOutOfBoundsException: 10
        scala.collection.LinearSeqOptimized.apply(Li
        nearSeqOptimized.scala:63)
        scala.collection.LinearSeqOptimized.apply$(L
        inearSeqOptimized.scala:61)
        scala.collection.immutable.List.apply(List.s
        cala:86)
        test.utest.FormatterTests$.$anonfun$tests$6(
        FormatterTests.scala:26)
    utest.asserts.Asserts$.assertImpl(Asserts.scala:114)
    test.utest.FormatterTests$.$anonfun$tests$10(FormatterTests.scala:160)
```

Scala PRs merged since 2.12.6, as shown by `git log --pretty=oneline --merges @ ^v2.12.6`:

```
bd76826ab8786ce6fd344b4d7eace972bf85cc3c (HEAD -> 2.12.x, origin/2.12.x) Merge pull request #6712 from exoego/scaladoc-placeholder-color
aae6a977aadc67b53e75972190327570185fbe03 Merge pull request #6439 from som-snytt/issue/10784
c1b68e4f4bf17be5aaaa2f008d271f26983096ca Merge pull request #6470 from DavidGregory084/bug#10808
11ef06f03048cef40ec3d3210c77a61058fd0a01 Merge pull request #6672 from hrhino/t10880
9e8a4215863db97eca7f3f18ae16a88931793934 Merge pull request #6655 from hrhino/t10886
c15a386e33fd7ba1b7c1b8635cf9d09d3c6c723f Merge pull request #6697 from lrytz/code-mono
32f194408428178d0b554f728596825007cd5ddc Merge pull request #6680 from lrytz/pr6370
d508e7df958baef838bd099939ce3317705c2963 Merge pull request #6479 from retronym/topic/compiler-interrupted
9136c5011b34cf010ad746c621dee4271c0e8ad6 Merge pull request #6365 from som-snytt/issue/8294
90c5264716c5967b33936b9227e0aba07b5030ce Merge pull request #6609 from som-snytt/issue/10869
e2a0bd992560b6901eab932276e86c0fa6e78e96 Merge pull request #6601 from Jasper-M/patch-2
c6172729129a07ff60a5ee9b1a19b6e59fcba376 Merge pull request #6586 from ashawley/2.12/clean-scaladoc-version
c3f53687d1fa1ac8ecc9cd9a6876cdcea9249ed2 Merge pull request #6555 from Jasper-M/topic/printFilter-backport
db73686aaf3780fb3a4dcbf354fc80b870aa47ef Merge pull request #6667 from som-snytt/issue/silent-noise
8a7e43922f59945fd890ffc411ff90edb9ab0c90 [nomerge] Merge pull request #6668 from som-snytt/issue/10892-it-strings
a473cb3e2d87fc9a1f48ff26e997797e65a6dd61 Merge pull request #6654 from hrhino/t10888
8bf17fcc7f3aedf5abf73e194f8f6310075c3635 Merge pull request #6455 from dskells/2.12.x_flag
d588c2a0092d9a22d200f7a91d6c8f58663b090a Merge pull request #6647 from Philippus/remove-scala-object
5331f0b41fa9ab93f46959919be2441d949598d2 Merge pull request #6630 from SethTisue/pr-validation-via-travis-mark-ii
a987b3815a9f8f0f8bdf414593cda6b9ada19e60 Merge pull request #6634 from GeorgiChochov/patch-1
534a92970801a5fc539d3002a5365d3f6f56cf3a Merge pull request #6613 from kamilduda/issue/10864
ae32078850f3ca29bb9954e800eded3e2bfea585 Merge pull request #6624 from SethTisue/allow-travis-pr-validation
3237e03fad62ace414b27e1cc2763bc72dcac294 Merge pull request #6560 from retronym/topic/reset-duplicate
7844a54169236c7b2ad758a1600e3c79faadf0ac Merge pull request #6529 from viktorklang/wip-10810-√
93ded124ca29a78d472421907ea0796dacfccab6 Merge pull request #6449 from retronym/ticket/SD-187
2beac5717ca91e29ff629b31be4d1fe7d8fa34fe Merge pull request #6585 from SethTisue/coc
10b364662b1c42e95c9c939129ddf4fe69361bd3 Merge pull request #6588 from SethTisue/no-sizzle
4d1236890e8664c03796c31a40be6365d5e39468 Merge pull request #6570 from SethTisue/bump-swing-version
689cd08d511d19c8396e238ac90e2c9ecadb20b0 Merge pull request #6149 from dragos/issue/8351-implicits-bounds-check
ed508872421e7e064806c52ab6467a5dcf2e71ab Merge pull request #6568 from SethTisue/restarr-onto-2.12.6
```

**Comment by SethTisue** [comment]
a bit of bisecting shows that the first red version is https://github.com/scala/scala/pull/6586 (`2.12.7-bin-c617272`) which seems mysterious since that PR seems innocuous

is it because `FormatterTests` is sensitive to the exact line numbers in a stack trace...?

**Comment by olafurpg** [comment]
Fixed in #194. The formatted tests no longer assert against line numbers from Scala-library
